src/main/java/edu/kit/ipd/crowdcontrol/objectservice/crowdworking/PlatformManager.java

```diff
@@ -52,12 +69,13 @@ public PlatformManager(List<Platform> crowdPlatforms, WorkerIdentification fallb
 
         //create hashmap of platforms
         platforms = crowdPlatforms.stream()
-                .collect(Collectors.toMap(Platform::getName, Function.identity()));
+                .collect(Collectors.toMap(Platform::getType, Function.identity()));
```

Comment:
I never knew about that, add another map with the type as key, and use it in identify worker (we have to write somewhere that the idenfitifaction should not differ in 2 instances...